src/routes/products/[product].svelte

```diff
@@ -47,4 +36,17 @@
 	.mzp-c-card-media-wrapper {
 		box-shadow: 0 0 0 2px $color-marketing-gray-20;
 	}
+	.dashboard {
+		display: block;
+		cursor: pointer;
+		&:hover {
```

Comment:
Another helpful hover effect is to darken the shadow a bit to emphasize the current card. Something like:

```css
box-shadow: 0 8px 12px 1px #1d11332a,0 3px 16px 2px #09204d1f,0 5px 10px -3px #1d11331f;
```

which just makes the existing shadow slightly less transparent. It looks like this would be a hover effect on `mzp-c-emphasis-box`, at least from my quick look at the deploy preview.